CMakeLists.txt

```diff
@@ -9,10 +9,13 @@ add_library(rabit src/allreduce_base.cc src/allreduce_robust.cc src/engine.cc sr
 add_library(rabit_base src/allreduce_base.cc src/engine_base.cc src/c_api.cc)
 add_library(rabit_empty src/engine_empty.cc src/c_api.cc)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
 set(rabit_libs rabit rabit_base rabit_empty)
 if(RABIT_BUILD_MPI)
   find_package(MPI REQUIRED)
   add_library(rabit_mpi src/engine_mpi.cc)
+  include_directories(SYSTEM ${MPI_INCLUDE_PATH})
```

Comment:
Like above comment.  But I'm ok with these old cmake script for now since it's not highly related to this PR.  But next time we can try sticking with those newer cmake style.  It will help making rabit a more portable project.

Reply:
fixed.